FATALDependency proj not found

I'm trying to build the lib using the following steps:

```
git clone https://github.com/intel/ad-rss-lib.git
cd ad-rss-lib
git submodule update --init
colcon build
```
However I'm getting

```
root@264371cba0f0:/home/ad-rss-lib# colcon build                           
Starting >>> spdlog  
Starting >>> ad_map_opendrive_reader
Starting >>> PROJ4
--- stderr: ad_map_opendrive_reader
FATALDependency proj not found!
CMake Error: The following variables are used in this project, but they are set to NOTFOUND.
Please set them or make sure they are set and tested correctly in the CMake files:
/home/ad-rss-lib/dependencies/map/ad_map_opendrive_reader/PROJ_INCLUDE_DIR
   used as include directory in directory /home/ad-rss-lib/dependencies/map/ad_map_opendrive_reader
PROJ_LIBRARIES
    linked by target "ad_map_opendrive_reader" in directory /home/ad-rss-lib/dependencies/map/ad_map_opendrive_reader

---
Failed   <<< ad_map_opendrive_reader [0.06s, exited with code 1]
Aborted  <<< spdlog [0.09s]
Aborted  <<< PROJ4 [0.14s]   
```

Could you advice if I am missing some preconditions.

Thank you!

--- a/colcon.meta
+++ b/colcon.meta
@@ -5,6 +5,12 @@
         },
         "spdlog": {
             "cmake-args": ["-DCMAKE_POSITION_INDEPENDENT_CODE=ON", "-DSPDLOG_BUILD_TESTS=OFF", "-DSPDLOG_BUILD_EXAMPLE=Off"]
-        }
+        },
+        "ad_map_opendrive_reader": {
+            "dependencies": ["PROJ4"]
+        },
+        "ad_map_access_qgis": {
+            "dependencies": ["ad_map_access"]
+        },
     }
 }

--- a/colcon_python.meta
+++ b/colcon_python.meta
@@ -8,12 +8,16 @@
         },
         "ad_map_opendrive_reader": {
             "cmake-args": ["-DBUILD_SHARED_LIBS=OFF", "-DCMAKE_POSITION_INDEPENDENT_CODE=ON", "-DCMAKE_BUILD_TYPE=RelWithDebInfo"]
+            "dependencies": ["PROJ4"]
         },
         "ad_rss": {
             "cmake-args": ["-DBUILD_SHARED_LIBS=OFF", "-DCMAKE_POSITION_INDEPENDENT_CODE=ON", "-DCMAKE_BUILD_TYPE=RelWithDebInfo", "-DBUILD_PYTHON_BINDING=ON"]
         },
         "ad_rss_map_integration": {
             "cmake-args": ["-DBUILD_SHARED_LIBS=OFF", "-DCMAKE_POSITION_INDEPENDENT_CODE=ON", "-DCMAKE_BUILD_TYPE=RelWithDebInfo", "-DBUILD_PYTHON_BINDING=ON"]
+        },
+        "ad_map_access_qgis": {
+            "dependencies": ["ad_map_access"]
         },
         "PROJ4": {
             "cmake-args": ["-DBUILD_SHARED_LIBS=OFF", "-DCMAKE_POSITION_INDEPENDENT_CODE=ON"]

--- a/colcon_python.meta
+++ b/colcon_python.meta
@@ -7,7 +7,7 @@
             "cmake-args": ["-DBUILD_SHARED_LIBS=OFF", "-DCMAKE_POSITION_INDEPENDENT_CODE=ON", "-DCMAKE_BUILD_TYPE=RelWithDebInfo", "-DBUILD_PYTHON_BINDING=ON"]
         },
         "ad_map_opendrive_reader": {
-            "cmake-args": ["-DBUILD_SHARED_LIBS=OFF", "-DCMAKE_POSITION_INDEPENDENT_CODE=ON", "-DCMAKE_BUILD_TYPE=RelWithDebInfo"]
+            "cmake-args": ["-DBUILD_SHARED_LIBS=OFF", "-DCMAKE_POSITION_INDEPENDENT_CODE=ON", "-DCMAKE_BUILD_TYPE=RelWithDebInfo"],
             "dependencies": ["PROJ4"]
         },
         "ad_rss": {

